lib/api-helper/machine/AbstractSoftwareDeliveryMachine.ts

```diff
@@ -261,6 +263,32 @@ export abstract class AbstractSoftwareDeliveryMachine<O extends SoftwareDelivery
             });
     }
 
+    /**
+     * Schedule the triggered listeners
+     */
+    public scheduleTriggeredListeners(): void {
+        const i: TriggeredListenerInvocation = {
+            addressAdmin: this.configuration.sdm.adminAddressChannels || (async msg => {
+                logger.info("trigger: %j", msg);
+            }),
+            sdm: this,
+        };
+
+        this.triggeredListeners.forEach(t => {
+            if (t.trigger && t.trigger.cron) {
+                const cron = new CronJob({
+                    cronTime: t.trigger.cron,
+                    onTick: () => t.listener(i),
+                    unrefTimeout: true,
+                } as any);
+                cron.start();
+            }
+            if (t.trigger && t.trigger.interval) {
```

Comment:
Could you have both cron and trigger? Docs indicate otherwise but this would allow it.

Reply:
Yes, you can have both. I'll update the comment to reflect that.